cmd/kpod/tag.go

```diff
@@ -60,19 +60,19 @@ func addImageNames(store storage.Store, image *storage.Image, addNames []string)
 }
 
 func expandedTags(tags []string) ([]string, error) {
+	var labelName string
 	expandedNames := []string{}
 	for _, tag := range tags {
-		name, err := reference.ParseNormalizedNamed(tag)
+		name, err := reference.Parse(tag)
 		if err != nil {
 			return nil, errors.Wrapf(err, "error parsing tag %q", name)
 		}
-
-		name = reference.TagNameOnly(name)
-		newTag := ""
-		if tagged, ok := name.(reference.NamedTagged); ok {
-			newTag = tagged.Tag()
+		if _, ok := name.(reference.NamedTagged); ok {
+			labelName = name.String()
+		} else {
+			labelName = name.String() + ":latest"
```

Comment:
I am probably not getting this right, but why is `:latest` being appended? What if the user wants to give their image a name without a tag like `myImage`?

Reply:
You *must* have a tag.  If the user does not provide a tag, the assumption then is latest.

Reply:
ok, gotcha. I thought the user can rename it to whatever they please.